electron-app/package.json

```diff
@@ -33,52 +33,52 @@
     }
   },
   "dependencies": {
-    "@theia/bulk-edit": "1.12.1",
-    "@theia/callhierarchy": "1.12.1",
-    "@theia/console": "1.12.1",
-    "@theia/core": "1.12.1",
-    "@theia/debug": "1.12.1",
-    "@theia/editor": "1.12.1",
-    "@theia/editor-preview": "1.12.1",
-    "@theia/electron": "1.12.1",
-    "@theia/external-terminal": "1.12.1",
-    "@theia/file-search": "1.12.1",
-    "@theia/filesystem": "1.12.1",
-    "@theia/getting-started": "1.12.1",
-    "@theia/git": "1.12.1",
-    "@theia/keymaps": "1.12.1",
-    "@theia/markers": "1.12.1",
-    "@theia/messages": "1.12.1",
-    "@theia/metrics": "1.12.1",
-    "@theia/mini-browser": "1.12.1",
-    "@theia/monaco": "1.12.1",
-    "@theia/navigator": "1.12.1",
-    "@theia/outline-view": "1.12.1",
-    "@theia/output": "1.12.1",
-    "@theia/plugin-dev": "1.12.1",
-    "@theia/plugin-ext": "1.12.1",
-    "@theia/plugin-ext-vscode": "1.12.1",
-    "@theia/preferences": "1.12.1",
-    "@theia/preview": "1.12.1",
-    "@theia/process": "1.12.1",
-    "@theia/property-view": "1.12.1",
-    "@theia/scm": "1.12.1",
-    "@theia/scm-extra": "1.12.1",
-    "@theia/search-in-workspace": "1.12.1",
-    "@theia/task": "1.12.1",
-    "@theia/terminal": "1.12.1",
-    "@theia/timeline": "1.12.1",
-    "@theia/typehierarchy": "1.12.1",
-    "@theia/userstorage": "1.12.1",
-    "@theia/variable-resolver": "1.12.1",
-    "@theia/vsx-registry": "1.12.1",
-    "@theia/workspace": "1.12.1",
+    "@theia/bulk-edit": "1.13.0",
+    "@theia/callhierarchy": "1.13.0",
+    "@theia/console": "1.13.0",
+    "@theia/core": "1.13.0",
+    "@theia/debug": "1.13.0",
+    "@theia/editor": "1.13.0",
+    "@theia/editor-preview": "1.13.0",
+    "@theia/electron": "1.13.0",
+    "@theia/external-terminal": "1.13.0",
+    "@theia/file-search": "1.13.0",
+    "@theia/filesystem": "1.13.0",
+    "@theia/getting-started": "1.13.0",
+    "@theia/git": "1.13.0",
```

Comment:
Can we remove `@theia/git` to fix https://github.com/eclipse-theia/theia-blueprint/issues/61, and prevent the clashing from the extension-pack?
The builtin also provides additional functionality like being able to use `@theia/timeline` among vscode git extensions like `gitlens`.

Reply:
done